app/controllers/provider_interface/application_choices_controller.rb

```diff
@@ -0,0 +1,31 @@
+module ProviderInterface
+  class ApplicationChoicesController < ProviderInterfaceController
+    def index
+      application_choices = ApplicationChoice
+        .includes(:application_form)
+        .where(provider_ucas_code: current_user.provider_ucas_code)
```

Comment:
should we move 
```
.includes(:application_form)
        .where(provider_ucas_code: current_user.provider_ucas_code
```
 to the model? 
since it is also present to the `show` action? 

(It can help in case we want to refactor all our queries in a Repo class for example)